internal/app/admin/http/testdata/lds_response.json

```diff
@@ -0,0 +1,34 @@
+{
+  "Cache": [
+    {
+      "Key": "lds",
+      "Resp": {
+        "VersionInfo": "1",
+        "Resources": {
+          "Endpoints": null,
+          "Clusters": null,
+          "Routes": null,
+          "Listeners": [
+            {
+              "name": "lds resource"
+            }
+          ],
+          "Secrets": null,
+          "Runtimes": null,
+          "Unmarshalled": null
+        },
+        "Canary": false,
+        "TypeURL": "type.googleapis.com/envoy.api.v2.Listener",
+        "Nonce": "",
+        "ControlPlane": null
+      },
+      "Requests": [
+        {
+          "version_info": "1",
+          "type_url": "type.googleapis.com/envoy.api.v2.Listener"
+        }
+      ],
+      "ExpirationTime": ""
+    }
+  ]
+}
```

Comment:
nit: newline

Reply:
i tried doing it, but the response doesnt match anymore because we do a strict check between filecontents and the response.